src/web/index.html

```diff
@@ -2,10 +2,12 @@
 <html lang="en">
   <head>
     <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
-    <title>code.pyret.org</title>
+    <title>Student Dashboard - code.pyret.org</title>
```

Comment:
I wonder if this should just say "Dashboard".  Not all users are students (I'm not, for example).  But all users get this index page/dashboard.